diff --git a/src/config.ts b/src/config.ts
index 097519f..76b0518 100644
--- a/src/config.ts
+++ b/src/config.ts
@@ -43,7 +43,6 @@ export const DEFAULT_FLOAT_RULES: Array<FloatRule> = [
     { class: "Steam", title: "^.*(Guard|Login).*" },
     { class: "TelegramDesktop", title: "Media viewer" },
     { class: "Zotero", title: "Quick Format Citation" },
-    { class: "firefox", title: "^(?!.*Mozilla Firefox).*$" },
     { class: "gjs" },
     { class: "gnome-screenshot" },
     { class: "ibus-.*" },
